config/logstash.yml

```diff
@@ -286,8 +286,17 @@
 # log.level: info
 # path.logs:
 #
-
-
+# ------------ Password Policy --------------
```

Comment:
Adding explanation that says:

This allows logstash to WARN or ERROR when the password set for basic HTTP authentication doesn't satisfy the password policy. The default it WARN, which will show a warning....

Reply:
@roaksoax Please take a look at [this PR](https://github.com/elastic/logstash/pull/14105] which simplifies configuration, better explanation mentions it is for HTTP basic auth usage and introduces default password policy. Thanks.